doc/usage/installation.rst

```diff
@@ -0,0 +1,147 @@
+=================
+Installing Sphinx
+=================
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+
+Linux
+-----
+
+Debian/Ubuntu
+~~~~~~~~~~~~~
+
+Install ``python-sphinx`` using :command:`apt-get`:
+
+.. code-block:: bash
+
+   $ apt-get install python-sphinx
+
+If it not already present, this will install Python for you.
+
+RHEL, CentOS
+~~~~~~~~~~~~
+
+Install ``python-sphinx`` using :command:`yum`:
+
+.. code-block:: bash
+
+   $ yum install python-sphinx
+
+If it not already present, this will install Python for you.
+
+Other distributions
+~~~~~~~~~~~~~~~~~~~
+
+Most Linux distributions have Sphinx in their package repositories.  Usually
+the package is called ``python-sphinx`` or ``sphinx``.  Be aware that there are
+at least two other packages with ``sphinx`` in their name: a speech recognition
+toolkit (*CMU Sphinx*) and a full-text search database (*Sphinx search*).
+
+
+macOS
+-----
+
+Sphinx can be installed using `MacPorts <http://www.macports.org/>`_.
+
+.. code-block:: bash
+
+   $ sudo port install py27-sphinx
+
+To set up the executable paths, use the ``port select`` command:
+
+.. code-block:: bash
+
+   $ sudo port select --set python python27
+   $ sudo port select --set sphinx py27-sphinx
+
+Type :command:`which sphinx-quickstart` to check if the installation was
+successful.
+
+
+Windows
+-------
+
+.. todo:: Could we start packaging this?
+
+Most Windows users do not have Python installed by default, so we begin with
+the installation of Python itself.  If you are unsure, open the *Command
+Prompt* (:kbd:`Control-r` and type :command:`cmd`).  Once the command prompt is
+open, type :command:`python --version` and press Enter.  If Python is
+available, you will see the version of Python printed to the screen.  If you do
+not have Python installed, refer to the *Hitchhikers Guide to Python*\'s Python
```

Comment:
Link to http://docs.python-guide.org/en/latest/ ?

Reply:
what do you think?

Reply:
Oh, I missed this. I think this might be confusing. I want users to click the Python 2 or Python 3 links and not the link to the main page. Thoughts?

Reply:
I guess the readers of this document will not notice the python2 or python3 links below is linked to hitchhikers guide because the link titles does not contains the page name.
So I think it is better to make this title linked or to arrange these link titles more understandable.

Reply:
Done